3main

```diff
@@ -155,26 +153,42 @@ P10X_100="NO"
 
 nvidia-smi -L > /tmp/tempa
 
-if grep -q "1050" /tmp/tempa;
-then
+if grep -q "1050" /tmp/tempa; then
   ___1050_or_1050ti="YES"
```

Comment:
actually the issue is not with TIs in general, but only with 1050 and 1050ti, which seems to be handled the same way as p10x cards